package.json

```diff
@@ -39,12 +39,12 @@
   },
   "dependencies": {
     "@endemolshinegroup/cosmiconfig-typescript-loader": "3.0.2",
-    "@graphql-tools/graphql-file-loader": "^7.3.2",
```

Comment:
Maybe just bump with keeping `^` to prevent duplication in node_modules otherwise graphql-config will need a release everything we release a new version of those graphql-tools packages.

Reply:
fixed, ty